### DISCUSSION TITLE
PullRequest Spam?

### DISCUSSION THREAD

**Discussion by pstorch** [OpenTracksApp/OpenTracks#1487] [General]
What is happening here? Lot's of PRs with many commits and resulting in merge conflicts.

**Comment by dennisguse** [comment]
I honestly don't know. 
Should we disable pull requests for now?

**Reply by pstorch** [reply]
Is that possible? If yes, then maybe for a few days until they realize it.

**Reply by dennisguse** [reply]
@pstorch could you activate this for one month?

https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository

**Reply by pstorch** [reply]
Done

**Reply by dennisguse** [reply]
Awesome!
Finally some silence in my mail account.
Thanks

Let's see what happens now.

**Reply by rgmf** [reply]
Great! It's been funny days 🤣

**Reply by dennisguse** [reply]
I just removed the interaction limitation on the repo - let‘s see how it goes the next days…
